### PR TITLE
update

### PR DESCRIPTION
**Pull request by mohsinenar**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM python:3.8-alpine
+RUN pip install ostorlab
+CMD ostorlab --api_key="$INPUT_OSTORLAB_API_KEY"  ci-scan  run  --title="$INPUT_SCAN_TITLE" --plan="$INPUT_PLAN"  --break_on_risk_rating="$INPUT_BREAK_ON_RISK_RATING" --max_wait_minutes="$INPUT_MAX_WAIT_MINUTES" $INPUT_ASSET_TYPE $INPUT_TARGET
```
Comment (amine3) [inline Dockerfile]: Missing log_flavor arg

File: README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Ostorlab ci_can for GithubAction
```
Comment (amine3) [inline README.md]: typo here and we can call it Ostorlab CI

File: README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Ostorlab ci_can for GithubAction
+![Ostorlab ci_can for GithubAction](https://i.ibb.co/XF3cwWw/image.png)
```
Comment (amine3) [inline README.md:2]: Same here

File: action.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
+name: ostorlab github action.
+description: ostorlab github action.
+inputs:
+  plan:
+    description: Specify which plan to use for scan.
+    required: true
+    default: free
```
Comment (amine3) [inline action.yml]: incorrect value, should be rapid_static

File: action.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  asset_type:
+    description: Type of scan android|ios.
```
Comment (amine3) [inline action.yml]: incorrect values

File: action.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    description: Path to file .apk|.ios file.
+    required: true
+  scan_title:
+    description: Title for your scan.
+    required: false
+  ostorlab_api_key:
+    description: Api key from ostorlab.
+    required: true
+  break_on_risk_rating:
+    description: Api key from ostorlab.
```
Comment (amine3) [inline action.yml]: incorrect description

**Review by amine3 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (mohsinenar) [inline Dockerfile]:
```suggestion
CMD ostorlab --api_key="$INPUT_OSTORLAB_API_KEY"  ci-scan  run  --title="$INPUT_SCAN_TITLE" --plan="$INPUT_PLAN"  --break_on_risk_rating="$INPUT_BREAK_ON_RISK_RATING" --max_wait_minutes="$INPUT_MAX_WAIT_MINUTES" $INPUT_ASSET_TYPE $INPUT_TARGET --log-flavor=github
```

File: action.yml (hunk 4 above)
Comment (mohsinenar) [inline action.yml]:
```suggestion
    default: rapid_static
```

File: action.yml (hunk 5 above)
Comment (mohsinenar) [inline action.yml:9]:
```suggestion
    description: Type of scan android-apk|android-aab|ios-ipa.
```

File: .github/workflows/action.yaml [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+      - name: Lunch Ostorlab scan
+        id: start_scan
+        uses: ./
+        with:
+          plan: rapid_static # Specify which plan to use for the scan (check plan section).
+          asset_type: android-apk # type of asset to scan.
+          target: ostorlab.apk # path for target tto scan.
+          scan_title: title_scan_ci # type a title for your scan.
+          ostorlab_api_key: ${{ secrets.ostorlab_api_key }} # your secret api key.
+          break_on_risk_rating: HIGH
```
Comment (3asm) [inline .github/workflows/action.yaml]: Since each field has comment, please add it to these as well.

File: Dockerfile [hunk 8]
```diff
@@ -0,0 +1,3 @@
+FROM python:3.8-alpine
+RUN pip install ostorlab
+CMD ostorlab --api_key="$INPUT_OSTORLAB_API_KEY"  ci-scan  run  --title="$INPUT_SCAN_TITLE" --plan="$INPUT_PLAN"  --break_on_risk_rating="$INPUT_BREAK_ON_RISK_RATING" --max_wait_minutes="$INPUT_MAX_WAIT_MINUTES" $INPUT_ASSET_TYPE $INPUT_TARGET --log-flavor=github
```
Comment (mohsinenar) [inline Dockerfile]:
```suggestion
CMD ostorlab  --api_key="$INPUT_OSTORLAB_API_KEY"  ci-scan  run --log-flavor=github --title="$INPUT_SCAN_TITLE" --plan="$INPUT_PLAN"  --break_on_risk_rating="$INPUT_BREAK_ON_RISK_RATING" --max_wait_minutes="$INPUT_MAX_WAIT_MINUTES" $INPUT_ASSET_TYPE $INPUT_TARGET
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+   	name: Test ostorlab ci actions.
+   	steps:
+   	 - uses: actions/checkout@v2
+   	 - name: Lunch Ostorlab scan
+   	   id: start_scan
+   	   uses: actions/ostorlab_actions@v1
+   		with:
+   		 plan: rapid_static 
+   		 asset_type: android-apk 
+   		 target: andoird_apk.apk
```
Comment (amine3) [inline README.md:47]: typo here, change it yo myApp.apk

File: README.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+   		 can_title: title_scan_ci
```
Comment (amine3) [inline README.md:48]: typo scan_title

**Review by amine3 — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (mohsinenar) [reply README.md:47]: myApp.apk
```suggestion
   		 target: myApp.apk

```

File: README.md (hunk 10 above)
Comment (mohsinenar) [inline README.md:48]:
```suggestion
   		 scan_title: title_scan_ci

```

File: README.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+   	steps:
+   	 - uses: actions/checkout@v2
+   	 - name: Lunch Ostorlab scan
+   	   id: start_scan
+   	   uses: actions/ostorlab_actions@v1
+   		with:
+   		 plan: rapid_static 
+   		 asset_type: android-apk 
+   		 target: myApp.apk
+
```
Comment (mohsinenar) [inline README.md]:
```suggestion
```

File: README.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+   		 scan_title: title_scan_ci
+
```
Comment (mohsinenar) [inline README.md]:
```suggestion
```

**Review by 3asm — APPROVED** [review]

**Pull request merged**